client/src/views/Settings/Tabs/DeleteAccount/index.vue

```diff
@@ -12,22 +12,51 @@
 
         <q-card-actions align="right">
           <q-btn flat label="Cancel" color="teal" v-close-popup />
-          <q-btn flat label="Delete" color="pink" v-close-popup />
+          <q-btn flat label="Delete" color="pink" @click="deleteProfile()" v-close-popup />
         </q-card-actions>
       </q-card>
     </q-dialog>
   </div>
 </template>
 
 <script>
+import { deleteAccount } from '@/services/profile.js'
+import { mapGetters } from 'vuex'
+
 export default {
   name: 'DeleteAccount',
   data () {
     return {
       confirm: false
     }
+  },
+  methods: {
+    deleteProfile () {
+      let user = this.getProfile
+      deleteAccount(user)
+        .then((res) => {
```

Comment:
Would be nice to handle errors if such occur

Reply:
done